src/command_modules/azure-cli-interactive/azclishell/az_completer.py

```diff
@@ -122,11 +125,13 @@ def __init__(self, commands, global_params=True, outstream=sys.stderr):
         self.global_parser = AzCliCommandParser(add_help=False)
         self.global_parser.add_argument_group('global', 'Global Arguments')
         self.parser = AzCliCommandParser(parents=[self.global_parser])
+        self.cmdtab = None
 
-        from azclishell._dump_commands import CMD_TABLE
-        self.cmdtab = CMD_TABLE
-        self.parser.load_command_table(CMD_TABLE)
-        self.argsfinder = ArgsFinder(self.parser, outstream)
+    def initialize_command_table_attributes(self):
+        self.cmdtab = FRESH_TABLE.command_table
+        if self.cmdtab is not None:
```

Comment:
What's special about the truthiness of `self.cmdtab`? Why not just `if self.cmdtab:`?